ext/reflection/php_reflection.stub.php

```diff
@@ -148,8 +150,8 @@ public function getExecutingGenerator() {}
 
 class ReflectionMethod extends ReflectionFunctionAbstract
 {
-    /** @param object|string $class_or_method */
-    public function __construct($class_or_method, string $name = UNKNOWN) {}
+    /** @param object|string $objectOrMethodName */
+    public function __construct($objectOrMethodName, string $methodName = UNKNOWN) {}
```

Comment:
The behaviour of `$objectOrMethodName` seems a bit crazy to me (it accepts either an object or a method name in the `Object::method` format). Should we deprecate it in the PHP 8.1 deprecations? And we could add a static `ReflectionMethod::fromMethodName(string $methodName)` method as a replacement. ping @Girgias @nikic

Reply:
That one seems pretty whack indeed, I think having the replacement method in PHP 8.0 would be ideal so that we can deprecate it in 8.1.

Reply:
Yes, exactly my thoughts too! I can provide it in a separate PR if it doesn't need any kind of "bureaucracy" ^^